launch/bullet_gazebo.launch

```diff
@@ -113,6 +113,6 @@
 
   <!-- convert joint states to TF transforms for rviz, etc -->
   <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" respawn="false" output="screen">
-    <remap from="/joint_states" to="/syropod/joint_states" />
+    <!-- <remap from="/joint_states" to="/syropod/joint_states" /> -->
```

Comment:
No need to preserve as a comment. Delete

Reply:
Deleted